examples/platforms/efr32mg21/radio.c

```diff
@@ -136,7 +136,7 @@ static const RAIL_IEEE802154_Config_t sRailIeee802154Config = {
     {
         // ackConfig
         true, // ackConfig.enable
-        894,  // ackConfig.ackTimeout
+        672,  // ackConfig.ackTimeout
```

Comment:
Why is this 672us? Acoording to my understanding:

For beacon-enabled PANs:
macAckWaitDuration = aUnitBackoffPeriod + aTurnaroundTime + phySHRDuration + 6 × phySymbolsPerOctet = 54 symbols = 864us

For beacon-disabled PANs:
macAckWaitDuration = aTurnaroundTime + phySHRDuration + 6 × phySymbolsPerOctet = 34 symbols = 544us

Reply:
Hi @dismirlian 

802.15.4 defines its macAckWaitDuration from the end of the transmitted packet to complete reception of the ACK. RAIL's ackTimeout only covers sync word detection of the ACK. Therefore,  we have to subtract the ACK's PHY header and payload time to get RAIL's ackTimeout setting. For 2.4 GHz OQPSK, macAckWaitDuration is specified as 54 symbols; subtracting 2-symbol PHY header and 10-symbol payload yields a RAIL ackTimeout of 42 symbols or 672 microseconds at 16 microseconds/symbol.

This has also been documented in the Silabs RAIL document here:
https://docs.silabs.com/rail/latest/group-i-e-e-e802-15-4

Reply:
https://docs.silabs.com/rail/latest/group-i-e-e-e802-15-4#details:~:text=802.15.4%20defines%20its%20macAckWaitDuration%20from%20the,or%20672%20microseconds%20at%2016%20microseconds%2Fsymbol.

Reply:
Great! Thanks for your answers. I see this note was added in RAIL's docs >= 2.8, but I assume it's valid for the previous versions also.

So the EFR32xG1x platform requires an update, right?

Reply:
Yes, we understand that there are inconsistencies between the different platform implementations. We will are planning to align them soon.